controllers/scopetemplate_controller.go

```diff
@@ -95,26 +122,31 @@ func (r *ScopeTemplateReconciler) Reconcile(ctx context.Context, req ctrl.Reques
 		// create ClusterRoles based on the ScopeTemplate
 		log.Log.Info("ScopeInstance found that references ScopeTemplate", "name", st.Name)
 		if err := r.ensureClusterRoles(ctx, st); err != nil {
-			return ctrl.Result{}, fmt.Errorf("Error in create ClusterRoles: %v", err)
+			return ctrl.Result{}, fmt.Errorf("creating ClusterRoles: %v", err)
 		}
+	}
 
-		// Add requirement to delete old hashes
-		requirement, err := labels.NewRequirement(scopeTemplateHashKey, selection.NotEquals, []string{util.HashObject(st.Spec)})
-		if err != nil {
-			return ctrl.Result{}, err
-		}
-		listOptions = append(listOptions, &client.ListOptions{
-			LabelSelector: labels.NewSelector().Add(*requirement),
-		})
-		break
+	// Add requirement to delete old hashes
```

Comment:
We aren't deleting old hashes, were deleting out of date (cluster)Roles.

Reply:
commit https://github.com/operator-framework/oria-operator/pull/48/commits/0d806dd453fda2c2a66e0739b7c3cf83500adea8